src/app/pages/categories/management/pages-transaction-categories.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+import { map, Observable, Subject } from 'rxjs';
+import { DomainsTransactionCategoriesGateway } from 'src/app/domains/categories/domains.transaction-categories.gateway';
+import { TransactionCategory } from '../transaction-category.model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class PagesTransactionCategoriesService {
+  private categories$: Subject<TransactionCategory[]> = new Subject<TransactionCategory[]>();
```

Comment:
I would use here `BehaviorSubject`. If you will emit data with `Subject` and then someone will subscribe to it you won't receive data until again it will be emitted. `BehaviorSubject` return always the last emitted data even if you will subscribe after emitting this data.